addons/mail/static/src/messaging/component/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -19,10 +19,11 @@ QUnit.module('ChatWindowManager', {
     beforeEach() {
         utilsBeforeEach(this);
         this.start = async params => {
-            let { env, widget } = await utilsStart(Object.assign({}, params, {
-                data: this.data,
+            let { env, widget } = await utilsStart(Object.assign({
                 hasChatWindow: true,
                 hasMessagingMenu: true,
+            }, params, {
+                data: this.data,
```

Comment:
what's the motivation for this change?

Reply:
To make sure `hasChatWindow` and `hasMessagingMenu` can be overriden by param on a test by test casis if necessary. Just making it more generic.